README.md

```diff
@@ -466,3 +466,17 @@ Babel will accept any PR for a proposal that is intended to go through the TC39
 It's our intention (and one of our main goals) as a project to help TC39 get feedback from the community through using the new syntax in their codebases. There is however a balance between supporting new proposals and informing users that any proposal is still just a proposal and subject to change. In order to move the community forward, we will continously make changes to adhere to the spec as patches. If bigger changes require a major version, we will deprecate older versions of plugins, document those changes, and see if we can automate the upgrade via a codemod. This way everyone is moving towards Stage 4 and closer to what will be in the spec if the proposal makes it to the end.
 
 This means we will also include proposals in this repo that haven't been presented to the committee yet if they are concrete enough and someone on TC39 is planning on presenting it (and marked appropriately).
+
+### What's the process?
+
+- Find a TC39 Champion
```

Comment:
Maybe the wording here could be made more specific, e.g.,

"Either choose a proposal which has been presented at TC39 or will be presented soon, or convince a TC39 member to champion a new proposal."